app/src/androidTestKiwix/java/org/kiwix/kiwixmobile/tests/DownloadTest.java

```diff
@@ -53,7 +53,8 @@
   public static void beforeClass() {
     IdlingPolicies.setMasterPolicyTimeout(350, TimeUnit.SECONDS);
     IdlingPolicies.setIdlingResourceTimeout(350, TimeUnit.SECONDS);
-    Espresso.registerIdlingResources(KiwixIdlingResource.getInstance());
+      //noinspection deprecation
+      Espresso.registerIdlingResources(KiwixIdlingResource.getInstance());
```

Comment:
This would mean we won't be notified in case the method gets deprecated. Right ? If yes, then it's not the intended outcome.